### ISSUE TITLE
Hook up new registration to rails app

### ISSUE THREAD

**Issue by rsgonzal** [Iridescent-CM/technovation-app#3024]

**Comment by viviancan** [comment]
https://mkdev.me/en/posts/rails-5-vue-js-how-to-stop-worrying-and-love-the-frontend

**Comment by viviancan** [comment]
Resources
- https://www.agiratech.com/create-application-rails-vue-js
- https://stackoverflow.com/questions/66275403/rails-back-end-and-vue-front-end-on-a-same-port

**Comment by viviancan** [comment]
- [x] Create endpoint to render the new registration form (/new-registration)

- [x] Transfer the tg-sandbox new registration code to technovation-app

Related tickets 
- #3033 

Update 9/2/21
I created an endpoint `/new-registration` which returns the new-registration form. The new registration uses Vue.js, tailwind css, and vue formulate. I added an additional ticket #3033 for the tailwind and formulate configuration. Some items still need to be cleaned up like the CSS, image rendring, and some poential refactoring
![Screen Shot 2021-09-02 at 11.29.42 AM.png](https://images.zenhubusercontent.com/606b0f4fd13d53dff3d14fa5/437c663c-47bf-4168-8db7-df422cb4ccd8)
